cmd/gctsCloneRepository.go

```diff
@@ -0,0 +1,101 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/cookiejar"
+
+	gabs "github.com/Jeffail/gabs/v2"
+	"github.com/SAP/jenkins-library/pkg/command"
+	piperhttp "github.com/SAP/jenkins-library/pkg/http"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/pkg/errors"
+)
+
+func gctsCloneRepository(config gctsCloneRepositoryOptions, telemetryData *telemetry.CustomData) {
+	// for command execution use Command
+	c := command.Command{}
```

Comment:
seems like no command is executed in this step, thus everything related to `command` could be removed